Add release and migration notes for selenium module changes (#278)
BSI-Business-Systems-Integration-AG/org.eclipse.scout.docs: docs/modules/migration/partials/migration-guide-25.1.adoc

TIP: Mobile browsers will still offer the possibility to add a home screen icon for the application (which will just be a bookmark, opened in the regular browser). To properly configure it as an "app", consider adding a https://developer.mozilla.org/en-US/docs/Web/Manifest[PWA manifest].

== Use scout properties for selenium test module configuration

> Test setup provided by module `org.eclipse.scout.rt.ui.html.selenium` uses now scout properties instead of system properties.

Reviewer: ```suggestion
Test setup provided by module `org.eclipse.scout.rt.ui.html.selenium` now uses scout properties instead of system properties.
```

> The following table contains the mapping (See also class `SeleniumProperties`).

Reviewer: ```suggestion
The following table contains the mapping (see also class `SeleniumProperties`).
```

[cols="2,1",options="header"]
|===
|old system property |scout config properties

|`web.app.url`
|`scout.selenium.webAppUrl`

|`query.params`
|`scout.selenium.webQueryParams`

|`take.screenshot.on.failure`
|`scout.selenium.screenshotOnFailure`

|`webdriver.chrome.driver`
|`scout.selenium.driverConfig[webdriver.chrome.driver]`

(now optional)

|new property
|`scout.selenium.driver`

Must be set either to `chrome`, `edge` or `firefox`.

|new property
|`scout.selenium.driverRemoteServerUrl`

May be set (usually to `http://localhost:4444`) in order to use selenium docker grid.

|===

> All driver internal configurations may be set with property `scout.selenium.driverConfig`.

Reviewer: ```suggestion
All driver-specific configurations may be set with the property `scout.selenium.driverConfig`.
```
